lib/src/data_table.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:collection/collection.dart';
+
+class DataTable {
+  const DataTable(this._data);
+
+  final List<List<dynamic>> _data;
+
+  List<List<dynamic>> asLists({bool ignoreFirstRow = false}) =>
```

Comment:
[This guide](https://www.baeldung.com/cucumber-data-tables) uses a table without headers and the [official documentation](https://github.com/cucumber/cucumber-jvm/tree/main/datatable) mentions that instead of the first raw, a table might have the first column as the keys.

Perhaps there might be cases where the data table is not organised or cases where the headers are redundant because the content of the table is self explanatory.

Reply:
Updated:
The flag was removed as it's not really needed.